### PR TITLE
Added samples for the Vibration API and NotificationOptions.vibrate

### PR DESCRIPTION
**Pull request by wibblymat**
I suspect that I haven't really followed best-practices here, so please feel free to nit-pick all you like.

R: @addyosmani @gauntface @jeffposnick

### CODE REVIEW

File: vibration/index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+feature_name: Vibration API
+chrome_version: 32
+feature_id: 5698768766763008
+---
+
+<p>You can control the vibration capability of your Android device using the Vibration API.</p>
+
+{% capture js1 %}
+function example1() {
+	// Vibrate for 500ms
```
Comment (jeffposnick) [inline vibration/index.html]: Two-space indentation throughout, please.

File: vibration/sw.js [hunk 2]
```diff
@@ -0,0 +1 @@
+self.skipWaiting();
```
Comment (jeffposnick) [inline vibration/sw.js]:
```
self.addEventListener('install', function(e) {
  e.waitUntil(self.skipWaiting());
});
```

File: vibration/notifications.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+chrome_version: 45
+feature_id: 5727996321202176
+---
+
+<p>When displaying a system notification your device may vibrate to alert the user. You can specify the vibration pattern that should be used in the same way as for the <a href="./index.html">Vibration API</a>.</p>
+
+<p>When creating the notification, just set the 'vibrate' property of the notification options to be a Vibration API pattern.</p>
+
+{% capture js %}
+var registrationPromise = navigator.serviceWorker.register('sw.js');
```
Comment (jeffposnick) [inline vibration/notifications.html]: Slight reworking:

```
navigator.serviceWorker.register('sw.js');

function showNotification() {
  Notification.requestPermission(function(result) {
    if (result === 'granted') {
      navigator.serviceWorker.ready.then(function(registration) {
        registration.showNotification('Vibration Sample', {
          body: 'Buzz! Buzz!',
          icon: '../images/touch/chrome-touch-icon-192x192.png',
          vibrate: [200, 100, 200, 100, 200, 100, 200],
          tag: 'vibration-sample'
        });
      });
    }
  });
}
```

The `navigator.serviceWorker.ready` promise will only resolve when there's an active SW, which more closely matches the intent. And this checks the result of the `requestPermission()` call.

File: vibration/notifications.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+feature_name: NotificationOptions.vibrate
+chrome_version: 45
+feature_id: 5727996321202176
+---
+
+<p>When displaying a system notification your device may vibrate to alert the user. You can specify the vibration pattern that should be used in the same way as for the <a href="./index.html">Vibration API</a>.</p>
+
+<p>When creating the notification, just set the 'vibrate' property of the notification options to be a Vibration API pattern.</p>
+
```
Comment (jeffposnick) [inline vibration/notifications.html:10]: Can you add a line of explanatory text about how you need to register a service worker in order to show a notification that makes use of the vibration option?

Comment (beverloo) [inline vibration/notifications.html:10]: This is a side-effect of us not supporting "new Notification()" on Android, which iOS and Firefox OS do support. Such notifications can support vibration just as well.

File: vibration/sw.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+self.addEventListener('install', function() {
```
Comment (beverloo) [inline vibration/sw.js:1]: self.onnotificationclick = e => e.notification.close();

**Comment by wibblymat** [comment]
Review comments addressed - PTAL.
@addyosmani @jeffposnick

**Pull request merged**